hack/cherry-pick-pull.sh

```diff
@@ -151,12 +152,14 @@ function make-a-pr() {
   local numandtitle
   numandtitle=$(printf '%s\n' "${SUBJECTS[@]}")
   relnotes=$(printf "${RELEASE_NOTES[@]}")
+  labels=$(printf "${LABELS[@]}")
   cat >"${prtext}" <<EOF
 [${rel}] Automated cherry pick of ${numandtitle}
 
 Cherry pick of ${PULLSUBJ} on ${rel}.
 
 ${numandtitle}
+${labels}
```

Comment:
Nit: is it possible to move the labels (if there are any) to the head of the PR description?

Reply:
Done.